response.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/monzo/terrors"
 )
 
+// A Request is Typhon's wrapper around http.Response, used by both clients and servers.
```

Comment:
```
15   1 warning  golint comment on exported type Response should be of the form "Response ..." (with optional leading article) (gometalinter)
```

Reply:
Ah shit, yes, that's a typo. I'll fix this.